retry.go

```diff
@@ -178,7 +178,7 @@ func sleepDuration(resp *Response, min, max time.Duration, attempt int) (time.Du
 		return jitterBackoff(min, max, attempt), nil
 	}
 
-	retryAfterFunc := resp.Request.client.RetryAfter
+	retryAfterFunc := resp.Request.client.retryAfter
```

Comment:
@tttturtle-russ Please use the newly created getter method instead of direct internal field access.
```go
retryAfterFunc := resp.Request.client.RetryAfter()
```